lib/Sisimai/Rhost/FrancePTT.pm

```diff
@@ -4,6 +4,7 @@ use strict;
 use warnings;
 
 my $ErrorCodes = {
+    '102' => 'spamdetected',  # 550 mwinf5c04 ME Adresse IP source bloquee pour incident de spam. Client host blocked for spamming issues. OFR006_102 Ref
```

Comment:
The value of error code `102` is "blocked" because this error message means "SMTP connection is rejected due client's IP address is listed in block list (sending many spams)"

Reply:
When an SMTP connection is rejected with a message like "spam is not allowed" at after sending SMTP DATA command or at checking message headers and body.